src/SingleMovie.js

```diff
@@ -1,18 +1,57 @@
 import React from 'react'
 import './SingleMovie.css'
+import * as apiCalls from './apiCalls'
 
-const SingleMovie = (props) => {
-    return (
-      <div>
-        <img className='single-movie'
-          src={props.displaySelectedMovie[0].poster_path} 
-        />
-        <button onClick={(e) => {
-          e.preventDefault()
-          props.returnHome()
-        }}>HOME</button>
+class SingleMovie extends React.Component {
+  constructor() {
+    super()
+    this.state= {
+      currentMoviePoster: null,
```

Comment:
Do you need to have all of these as a state of your component? How can you DRY this state up?